modules/ROOT/pages/administration/aliases.adoc

```diff
@@ -283,6 +283,18 @@ The required privileges are described xref::administration/access-control/dbms-a
 
 | database
 | The name of the target database. label:default-output[]
+
+This column is filtered according to the `ACCESS` privileges of the user.
+However, some privileges enable users to see additional databases regardless of their `ACCESS` privileges:
+
+* Users with `CREATE/DROP/ALTER DATABASE` or `SET DATABASE ACCESS` privileges can see all standard databases.
```

Comment:
```suggestion
* Users with the `CREATE DATABASE`, `DROP DATABASE`, `ALTER DATABASE`, or `SET DATABASE ACCESS` privileges can see all standard databases.
```

Reply:
If we change this here we should probably change it in the database chapter as well as it has the same text